src/madhuraMJun20/scannerClass/PalindromeNumber.java

```diff
@@ -0,0 +1,32 @@
+package madhuraMJun20.scannerClass;
+
+/*Verify given number is palindrome or not. 
+Input: number = 45654
+Output : Given number 45654 is palindrome.*/
+public class PalindromeNumber {
+	int reverseNum;
+
+	void findPalindromeNum(int number) {
+		int mod = 0;
+		while (number > 0) {
+			mod = number % 10;      // mod= mod *10 + number% 10; number = number/10;
+			number = number / 10;
+			reverseNum = mod + reverseNum * 10;
+		}
+	}
+
+	void display(int number) {
```

Comment:
Why you have created extra method when you have already given method name as findPalindromeNum(num).
As per method name it looks like after calling this method I got the answer.